freqtrade/optimize/backtesting.py

```diff
@@ -567,6 +609,8 @@ def handle_left_open(self, open_trades: Dict[str, List[LocalTrade]],
         for pair in open_trades.keys():
             if len(open_trades[pair]) > 0:
                 for trade in open_trades[pair]:
+                    if trade.open_order_id:
+                        continue
```

Comment:
@rokups what's the intend with this?

assuming the open_order_id is a sell order that's not yet closed - this would "hide" that trade by not appending it to the "left open" trades list...

Reply:
This was meant to get rid of trades that are pending a buy but havent been filled

Reply:
that'll only work in some conditions (if it's a "initial" buy), but will cause odd results in case of open sell orders - as well as when it's a "2nd" buy.